web/themes/custom/cloudy/templates/content/alert/node--alert--featured.html.twig

```diff
@@ -1,6 +1,4 @@
-{% embed 'node--alert.html.twig' with {
-  'dismissible': TRUE
+{% include 'node--alert.html.twig' with {
+  dismissible: TRUE,
+  showTimestamp: FALSE,
```

Comment:
@rockymountainhigh1943, possible quick fix. The template has showTimestamp set as false, but it is still showing for the featured display mode. I was surprised to see that the teaser and feature were inheriting from node--alert.html.twig instead of the component. I thought all the display modes would pull in the component with specific configuration. That may be a misunderstanding of how it should work on my part.